src/common/SurgeStorage.cpp

```diff
@@ -371,7 +373,8 @@ void SurgeStorage::load_wt_wt(string filename, Wavetable* wt)
       ds = sizeof(float) * vt_read_int16LE(wh.n_tables) * vt_read_int32LE(wh.n_samples);
 
    data = malloc(ds);
-   fread(data, 1, ds, f);
+   if (fread(data, 1, ds, f) != ds)
+      fprintf(stderr, "SurgeStorage: reading the wavetable data failed.\n");
```

Comment:
Same here, is fprintf(err) enough?

Reply:
`fprintf()` does not set `error`. You query for EOF with `feof()` and _some_ error with `ferror()` but it doesn't give any better granularity. Also here I would suggest updating the patch with FIXME comment.